tests/integration/validatorTests.py

```diff
@@ -39,55 +40,48 @@ def setUpClass(cls):
         column1.required = True
         column1.field_type = stringType
         column1.file_id = 1
-        cls.interfaces.validationDb.populateFile(column1)
```

Comment:
What this method did was use FileColumn.file_id to populate the FileColumn.file relationship (which holds the file type), this would normally be populated by doing a joinedload, but here the FileColumn objects are being built manually.  The ValidatorTests setup populates the file type relationship for each of the columns, these column definitions get passed in to the validate function, I'm not sure how the file type is used within that code.